### PR TITLE
fx: add FxService definitions

### PR DESCRIPTION
**Pull request by jwangsadinata**
This commit adds the foreign exchange service definitions to create,
update and retrieve a foreign exchange quote.

### CODE REVIEW

File: v1/fx/fx.gunk [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"FX"},
+	//         Description: "Retrieve the exchange rate from a currency to another",
+	//         Summary:     "Retrieve exchange rate between two currencies",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/fxFX",
```
Comment (nkristianto) [inline v1/fx/fx.gunk:121]: i think this should be `"#/definitions/fxGetCurrentFCRateResonse",`

Comment (ghazninattarshah) [reply v1/fx/fx.gunk:121]: the response type is `FX`, there is no `GetCurrentFCRateResonse` defined.

File: v1/fx/fx.gunk [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"FX"},
+	//         Description: "Creates a new foreign exchange quote and returns it.",
+	//         Summary:     "Create a foreign exchange quote",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "201": openapiv2.Response{
+	//                         Description: "Foreign Exchange created successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/fxCreateFX",
```
Comment (ghazninattarshah) [inline v1/fx/fx.gunk]: change it to `/fxFX`

**Review by ghazninattarshah — APPROVED** [review]
LGTM

File: v1/fx/fx.gunk (hunk 1 above)
Comment (nkristianto) [reply v1/fx/fx.gunk:121]: whoops, you;re right. the response is FX. my bad

File: v1/fx/fx.gunk (hunk 2 above)
Comment (jwangsadinata) [reply v1/fx/fx.gunk]: thanks @ghazninattarshah, fixed.

**Review by favadi — APPROVED** [review]

**Review by nkristianto — APPROVED** [review]

**Pull request merged**